lib/src/utils/constants.dart

```diff
@@ -1,7 +1,16 @@
 abstract class Constants {
   static const String apiKey = 'a1f0fb356abe09082c8e841e096b8dde';
-  static const String apiFetchUrl =
+  static const String apiFetchPopular =
       'https://api.themoviedb.org/3/movie/popular?api_key=';
+  static const String apiFetchSearch =
+      'https://api.themoviedb.org/3/search/movie?api_key=';
+  static const String apiSearchParam = '&query=';
```

Comment:
Separate all relative to base url used in provider and api into a new file.

Reply:
Done